src/main/java/core/basesyntax/dao/FruitShopDaoImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+import java.util.Optional;
+
+public class FruitShopDaoImpl implements FruitShopDao {
```

Comment:
rename FruitShopDaoMapImpl as you are using map as db

Reply:
agree, fixed